3cn-ecn/nantralPlatform
# React app calendar multiple events

# Description

Implement the display when events occur in same time in the calendar.
The events blocks take place at the right height and take the most width possible.
(The blocks have only been rightfully placed, their display is not very pretty at the moment).

# Tests

1. Steps to reproduce :

- Create some events in the week of 5-11 June
- Try to do some of them with common time area
- Go to the calendar in the event page

2. Expected results :

- The events appear at a good place on the calendar

# Screenshots

![image](https://user-images.githubusercontent.com/97662746/216397667-c6b4eab7-0626-4274-be0c-26aa185171b9.png)
